web-app/react-app/utils/web3Provider.js

```diff
@@ -15,8 +16,13 @@ export class StreamrWeb3 extends Web3 {
     isEnabled = (): boolean => !!this.currentProvider
```

Comment:
Should we also change the way this function works? Or just let it be / remove it

Reply:
This can be left as is.